core/generator.js

```diff
@@ -113,13 +118,25 @@ Blockly.Generator.prototype.workspaceToCode = function(workspace) {
       code.push(line);
     }
   }
-  code = code.join('\n');  // Blank line between each section.
-  code = this.finish(code);
-  // Final scrubbing of whitespace.
-  code = code.replace(/^\s+\n/, '');
-  code = code.replace(/\n\s+$/, '\n');
-  code = code.replace(/[ \t]+\n/g, '\n');
-  return code;
+  if(blocksNotGenerated.length == 0){
+    code = code.join('\n');  // Blank line between each section.
+    code = this.finish(code);
+    // Final scrubbing of whitespace.
+    code = code.replace(/^\s+\n/, '');
+    code = code.replace(/\n\s+$/, '\n');
+    code = code.replace(/[ \t]+\n/g, '\n');
+    return code;
+  }
+  else{
+    var textToBeLogged = 'The generator code for the following blocks not specified for ' + this.name_ + ': \n';
```

Comment:
Google style:
 * `else` on the same line as both curly braces, with one space on each side: `} else {`
 * 80 character line length. You may need to break this into multiple concatenated strings.